Production 2022-05-25

lazysoundsystem opened the pull request.

lazysoundsystem committed:
chore: change fallback language for ukr

Refs: DSR-388, DSR-374

lazysoundsystem committed:
chore: revert fallback change and add month names for ukr

Refs: DSR-388, DSR-374

lazysoundsystem committed:
chore: try alternative fallback for ukr

Refs: DSR-388

lazysoundsystem committed:
chore: revert fallback language for ukr to en

Refs: DSR-388

lazysoundsystem committed:
Merge pull request #423 from UN-OCHA/DSR-388-fallback-language-for-ukr

chore: change fallback language for ukr

lazysoundsystem committed:
chore(release): 2.10.9

Refs: DSR-388

lazysoundsystem committed:
chore(release): 2.10.9

Refs: DSR-388

lazysoundsystem committed:
fix merge conflict

lazysoundsystem committed:
Merge pull request #424 from UN-OCHA/DSR-388-fallback-language-for-ukr

Dsr 388 fallback language for ukr

dependabot[bot] committed:
build(deps): bump jsdom from 11.12.0 to 16.7.0

Bumps [jsdom](https://github.com/jsdom/jsdom) from 11.12.0 to 16.7.0.
- [Release notes](https://github.com/jsdom/jsdom/releases)
- [Changelog](https://github.com/jsdom/jsdom/blob/master/Changelog.md)
- [Commits](https://github.com/jsdom/jsdom/compare/11.12.0...16.7.0)

---
updated-dependencies:
- dependency-name: jsdom
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

lazysoundsystem committed:
Merge pull request #426 from UN-OCHA/dependabot/npm_and_yarn/jsdom-16.7.0

build(deps): bump jsdom from 11.12.0 to 16.7.0

dependabot[bot] committed:
build(deps): bump parse-url from 6.0.0 to 6.0.2

Bumps [parse-url](https://github.com/IonicaBizau/parse-url) from 6.0.0 to 6.0.2.
- [Release notes](https://github.com/IonicaBizau/parse-url/releases)
- [Commits](https://github.com/IonicaBizau/parse-url/commits)

---
updated-dependencies:
- dependency-name: parse-url
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

lazysoundsystem committed:
Merge pull request #427 from UN-OCHA/dependabot/npm_and_yarn/parse-url-6.0.2

build(deps): bump parse-url from 6.0.0 to 6.0.2

dependabot[bot] committed:
build(deps): bump terser from 4.8.0 to 4.8.1

Bumps [terser](https://github.com/terser/terser) from 4.8.0 to 4.8.1.
- [Release notes](https://github.com/terser/terser/releases)
- [Changelog](https://github.com/terser/terser/blob/master/CHANGELOG.md)
- [Commits](https://github.com/terser/terser/commits)

---
updated-dependencies:
- dependency-name: terser
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

lazysoundsystem committed:
Merge pull request #428 from UN-OCHA/dependabot/npm_and_yarn/terser-4.8.1

build(deps): bump terser from 4.8.0 to 4.8.1

dependabot[bot] committed:
build(deps): bump moment from 2.29.2 to 2.29.4

Bumps [moment](https://github.com/moment/moment) from 2.29.2 to 2.29.4.
- [Release notes](https://github.com/moment/moment/releases)
- [Changelog](https://github.com/moment/moment/blob/develop/CHANGELOG.md)
- [Commits](https://github.com/moment/moment/compare/2.29.2...2.29.4)

---
updated-dependencies:
- dependency-name: moment
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

lazysoundsystem committed:
Merge pull request #429 from UN-OCHA/dependabot/npm_and_yarn/moment-2.29.4

build(deps): bump moment from 2.29.2 to 2.29.4

rupl reviewed (commented):
I don't see the 2.10.10 release in dev yet, but otherwise looks good.

dependabot[bot] committed:
build(deps): bump moment-timezone from 0.5.34 to 0.5.37

Bumps [moment-timezone](https://github.com/moment/moment-timezone) from 0.5.34 to 0.5.37.
- [Release notes](https://github.com/moment/moment-timezone/releases)
- [Changelog](https://github.com/moment/moment-timezone/blob/develop/changelog.md)
- [Commits](https://github.com/moment/moment-timezone/compare/0.5.34...0.5.37)

---
updated-dependencies:
- dependency-name: moment-timezone
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

lazysoundsystem committed:
Merge pull request #431 from UN-OCHA/dependabot/npm_and_yarn/moment-timezone-0.5.37

build(deps): bump moment-timezone from 0.5.34 to 0.5.37

dependabot[bot] committed:
build(deps): bump loader-utils from 1.4.0 to 1.4.1

Bumps [loader-utils](https://github.com/webpack/loader-utils) from 1.4.0 to 1.4.1.
- [Release notes](https://github.com/webpack/loader-utils/releases)
- [Changelog](https://github.com/webpack/loader-utils/blob/v1.4.1/CHANGELOG.md)
- [Commits](https://github.com/webpack/loader-utils/compare/v1.4.0...v1.4.1)

---
updated-dependencies:
- dependency-name: loader-utils
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

lazysoundsystem committed:
Merge pull request #432 from UN-OCHA/dependabot/npm_and_yarn/loader-utils-1.4.1

build(deps): bump loader-utils from 1.4.0 to 1.4.1

dependabot[bot] committed:
build(deps): bump loader-utils from 1.4.1 to 1.4.2

Bumps [loader-utils](https://github.com/webpack/loader-utils) from 1.4.1 to 1.4.2.
- [Release notes](https://github.com/webpack/loader-utils/releases)
- [Changelog](https://github.com/webpack/loader-utils/blob/v1.4.2/CHANGELOG.md)
- [Commits](https://github.com/webpack/loader-utils/compare/v1.4.1...v1.4.2)

---
updated-dependencies:
- dependency-name: loader-utils
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

lazysoundsystem committed:
Merge pull request #433 from UN-OCHA/dependabot/npm_and_yarn/loader-utils-1.4.2

build(deps): bump loader-utils from 1.4.1 to 1.4.2

dependabot[bot] committed:
build(deps): bump decode-uri-component from 0.2.0 to 0.2.2

Bumps [decode-uri-component](https://github.com/SamVerschueren/decode-uri-component) from 0.2.0 to 0.2.2.
- [Release notes](https://github.com/SamVerschueren/decode-uri-component/releases)
- [Commits](https://github.com/SamVerschueren/decode-uri-component/compare/v0.2.0...v0.2.2)

---
updated-dependencies:
- dependency-name: decode-uri-component
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

lazysoundsystem committed:
Merge pull request #434 from UN-OCHA/dependabot/npm_and_yarn/decode-uri-component-0.2.2

build(deps): bump decode-uri-component from 0.2.0 to 0.2.2

dependabot[bot] committed:
build(deps): bump json5 from 1.0.1 to 1.0.2

Bumps [json5](https://github.com/json5/json5) from 1.0.1 to 1.0.2.
- [Release notes](https://github.com/json5/json5/releases)
- [Changelog](https://github.com/json5/json5/blob/main/CHANGELOG.md)
- [Commits](https://github.com/json5/json5/compare/v1.0.1...v1.0.2)

---
updated-dependencies:
- dependency-name: json5
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

lazysoundsystem committed:
Merge pull request #435 from UN-OCHA/dependabot/npm_and_yarn/json5-1.0.2

build(deps): bump json5 from 1.0.1 to 1.0.2

dependabot[bot] committed:
build(deps): bump luxon from 1.28.0 to 1.28.1

Bumps [luxon](https://github.com/moment/luxon) from 1.28.0 to 1.28.1.
- [Release notes](https://github.com/moment/luxon/releases)
- [Changelog](https://github.com/moment/luxon/blob/master/CHANGELOG.md)
- [Commits](https://github.com/moment/luxon/compare/1.28.0...1.28.1)

---
updated-dependencies:
- dependency-name: luxon
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

lazysoundsystem committed:
Merge pull request #436 from UN-OCHA/dependabot/npm_and_yarn/luxon-1.28.1

build(deps): bump luxon from 1.28.0 to 1.28.1

dependabot[bot] committed:
build(deps): bump ua-parser-js from 0.7.31 to 0.7.33

Bumps [ua-parser-js](https://github.com/faisalman/ua-parser-js) from 0.7.31 to 0.7.33.
- [Release notes](https://github.com/faisalman/ua-parser-js/releases)
- [Changelog](https://github.com/faisalman/ua-parser-js/blob/master/changelog.md)
- [Commits](https://github.com/faisalman/ua-parser-js/compare/0.7.31...0.7.33)

---
updated-dependencies:
- dependency-name: ua-parser-js
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

lazysoundsystem committed:
Merge pull request #437 from UN-OCHA/dependabot/npm_and_yarn/ua-parser-js-0.7.33

build(deps): bump ua-parser-js from 0.7.31 to 0.7.33

rupl committed:
fix(fts): remove 2019, add 2023

Refs: DSR-392

rupl committed:
fix(fts): define variable before using it

Refs: DSR-392

rupl committed:
chore: update dependencies after running yarn locally

rupl committed:
fix(fts): guard against missing data

Refs: DSR-392

rupl committed:
test: linting error

rupl committed:
Merge pull request #438 from UN-OCHA/DSR-392-fts-2023-autobuild

Refs: DSR-392

dependabot[bot] committed:
build(deps): bump json5 from 1.0.1 to 1.0.2

Bumps [json5](https://github.com/json5/json5) from 1.0.1 to 1.0.2.
- [Release notes](https://github.com/json5/json5/releases)
- [Changelog](https://github.com/json5/json5/blob/main/CHANGELOG.md)
- [Commits](https://github.com/json5/json5/compare/v1.0.1...v1.0.2)

---
updated-dependencies:
- dependency-name: json5
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump async from 2.6.3 to 2.6.4

Bumps [async](https://github.com/caolan/async) from 2.6.3 to 2.6.4.
- [Release notes](https://github.com/caolan/async/releases)
- [Changelog](https://github.com/caolan/async/blob/v2.6.4/CHANGELOG.md)
- [Commits](https://github.com/caolan/async/compare/v2.6.3...v2.6.4)

---
updated-dependencies:
- dependency-name: async
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

rupl committed:
Merge pull request #439 from UN-OCHA/dependabot/npm_and_yarn/json5-1.0.2

build(deps): bump json5 from 1.0.1 to 1.0.2

rupl committed:
Merge pull request #441 from UN-OCHA/dependabot/npm_and_yarn/async-2.6.4

build(deps): bump async from 2.6.3 to 2.6.4

rupl committed:
chore(release): 2.10.10

rupl committed:
Merge pull request #442 from UN-OCHA/release-2023-03-21

chore(release): 2.10.10

teodorescuserban committed:
[OPS-8820] remove New Relic.

rupl committed:
chore: remove newrelic gitignore rule

Refs: OPS-8820

rupl committed:
Merge pull request #443 from UN-OCHA/ops-8820

[OPS-8820] remove New Relic.

dependabot[bot] committed:
build(deps): bump semver from 5.7.1 to 5.7.2

Bumps [semver](https://github.com/npm/node-semver) from 5.7.1 to 5.7.2.
- [Release notes](https://github.com/npm/node-semver/releases)
- [Changelog](https://github.com/npm/node-semver/blob/v5.7.2/CHANGELOG.md)
- [Commits](https://github.com/npm/node-semver/compare/v5.7.1...v5.7.2)

---
updated-dependencies:
- dependency-name: semver
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

rupl committed:
Merge pull request #445 from UN-OCHA/dependabot/npm_and_yarn/semver-5.7.2

build(deps): bump semver from 5.7.1 to 5.7.2

dependabot[bot] committed:
build(deps): bump word-wrap from 1.2.3 to 1.2.4

Bumps [word-wrap](https://github.com/jonschlinkert/word-wrap) from 1.2.3 to 1.2.4.
- [Release notes](https://github.com/jonschlinkert/word-wrap/releases)
- [Commits](https://github.com/jonschlinkert/word-wrap/compare/1.2.3...1.2.4)

---
updated-dependencies:
- dependency-name: word-wrap
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

lazysoundsystem committed:
Merge pull request #446 from UN-OCHA/dependabot/npm_and_yarn/word-wrap-1.2.4

build(deps): bump word-wrap from 1.2.3 to 1.2.4

lazysoundsystem committed:
chore: update node-js version to 20

Refs: OPS-9554

lazysoundsystem committed:
chore: adjust node version in tests too

Refs: OPS-9554

rupl committed:
chore: keep nvmrc up to date

Refs: OPS-9554

rupl committed:
chore: autobuild when branchname begins with OPS

rupl committed:
fix: get rid of local development errors on a fresh node_modules install

Refs: OPS-9554

rupl committed:
Merge pull request #449 from UN-OCHA/OPS-9554-code-updates

[OPS-9554] code updates

rupl committed:
Merge pull request #448 from UN-OCHA/OPS-9554-node-20

chore: update node-js version to 20

lazysoundsystem commented:
I think this PR has passed it's EOL. Closing.
